settings.gradle

```diff
@@ -1,4 +1,21 @@
+// TODO: Remove once Spotless 3.17.0 is published to Gradle plugin portal
+pluginManagement {
```

Comment:
For consideration, it could be useful to add a link to your [above comment](https://github.com/triplea-game/triplea/pull/4569#discussion_r247326720} @ssoloff   to the source code as part of the TODO comment.